GOVERNANCE.md

```diff
@@ -0,0 +1,46 @@
+# Process for becoming a maintainer
+
+## Your organization is not yet a maintainer
+
+* Express interest to the senior maintainers that your organization is interested in becoming a maintainer. Becoming a maintainer generally means that you are going to be spending substantial time (>25%) on Amundsen for the foreseeable future. Ultimately your goal is to become a senior maintainer that will represent your organization.
+* We will expect you to start contributing increasingly complicated PRs, under the guidance of the existing senior maintainers.
+* We may ask you to do some PRs from our backlog.
+* As you gain experience with the code base and our standards, we will ask you to do code reviews for incoming PRs (i.e., all maintainers are expected to shoulder a proportional share of community reviews).
+* After a period of approximately 2-3 months of working together and making sure we see eye to eye, the existing senior maintainers will confer and decide whether to grant maintainer status or not. We make no guarantees on the length of time this will take, but 2-3 months is the approximate goal.
+
+## Your organization is currently a maintainer
+
+* First decide whether your organization really needs more people with maintainer access. Valid reasons are "blast radius", a large organization that is working on multiple unrelated projects, etc.
+* Contact a senior maintainer for your organization and express interest.
```

Comment:
What if your organization doesn't yet have a senior maintainer? Also, How would one know who is a senior maintainer and who is not? And how would they contact them? Is there a formal process to contact or can someone just DM directly on slack?

Reply:
I think those are legit questions.